src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/controllers/CaseActionsForCaseWorkerController.java

```diff
@@ -278,6 +278,7 @@ public ResponseEntity<CCDCallbackResponse> postDefaultValues(
             //create NOC answers section
             caseData = nocRespondentRepresentativeService.prepopulateOrgPolicyAndNoc(caseData);
             caseFlagsService.setupCaseFlags(caseData);
+            caseFlagsService.setDefaultFlags(caseData);
```

Comment:
unit test please

Reply:
this already exists

Reply:
setDefaultFlags still not being verified in your test